config/environments/production.rb

```diff
@@ -71,7 +72,7 @@
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
   # the I18n.default_locale when a translation cannot be found).
-  config.i18n.fallbacks = [I18n.default_locale]
```

Comment:
Toto asi chceme nechat ako bolo ci?

Reply:
ono je to to iste
> You can set the option to true for using default locale as fallback

https://guides.rubyonrails.org/v7.0.8/configuring.html#config-i18n-fallbacks